src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,17 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private final User[] users = new User[] {
```

Comment:
Don`t change it

Reply:
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/63095184/63655cda-23b0-43df-9e53-e9ba0c03313e)
Here is marked as static

Reply:
I removed static a long time ago. I don't understand why you don't see it